src/Apps/Order/Mutations/SetPaymentByStripeIntent.ts

```diff
@@ -0,0 +1,52 @@
+import { commitMutation, Environment, graphql } from "relay-runtime"
+import {
+  SetPaymentByStripeIntentMutation,
+  CommerceSetPaymentByStripeIntentInput,
+  SetPaymentByStripeIntentMutationResponse,
+} from "__generated__/SetPaymentByStripeIntentMutation.graphql"
+
+export const SetPaymentByStripeIntent = (
```

Comment:
This mutation looks good, but can we use our `useMutation` hook and follow this pattern? (And try to spread the word :) https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/DeleteAccount/useDeleteAccount.ts